utils/manifest-utils/src/keystore/x509_keystore.rs

```diff
@@ -48,8 +49,6 @@ impl X509CertData {
         add_cert_subject_entries(&mut subject, cert, Nid::PKCS9_EMAILADDRESS, "E");
         add_cert_subject_entries(&mut subject, cert, Nid::ORGANIZATIONNAME, "O");
         add_cert_subject_entries(&mut subject, cert, Nid::ORGANIZATIONALUNITNAME, "OU");
-        add_cert_subject_entries(&mut subject, cert, Nid::COUNTRYNAME, "C");
-        add_cert_subject_entries(&mut subject, cert, Nid::STATEORPROVINCENAME, "ST");
```

Comment:
Deleted this fields because:
- adding column `Type` make `keystore list` output a bit wide
- we will  add keystore cmd to display details of certificates
Also probably `Subject` value for keystore at some point will change so it will show only common name, or organization name to make it look similar to what is shown for Golem certificate.